app/models/attachment.rb

```diff
@@ -2,4 +2,18 @@ class Attachment < ActiveRecord::Base
   mount_uploader :file_upload, FileUploader
 
   belongs_to :attachable, polymorphic: true
+
+  # Download url of the file
```

Comment:
`file_url` will return a url (with host) if file is in remote server, otherwise a relative path without host.
`file_path` will  give you the absolute path of the file.